src/ucp/proto/proto_select.c

```diff
@@ -19,6 +19,10 @@
 #include <ucp/core/ucp_worker.inl>
 #include <ucs/datastruct/array.inl>
 
+__KHASH_IMPL(ucp_proto_select_hash, kh_inline,  khint64_t,
```

Comment:
Is that intentional to change `KHASH_TYPE`/`KHASH_IMPL` to `KHASH_DECLARE`/`__KHASH_IMPL`? Seems now `proto_select.inl` would not inline khash functions anymore?

Reply:
right, reverted in place (force pushed), because the patch is small.